spark/src/test/scala/org/apache/spark/sql/delta/test/DeltaTestImplicits.scala

```diff
@@ -181,6 +182,12 @@ object DeltaTestImplicits {
     def snapshot: Snapshot = deltaTable.initialSnapshot
   }
 
+  implicit class TahoeLogFileIndexObjectTestHelper(index: TahoeLogFileIndex.type) {
+    def apply(spark: SparkSession, deltaLog: DeltaLog): TahoeLogFileIndex = {
```

Comment:
Are there existing tests which uses the 2 arg constructor?

Reply:
Yes. There are quite a few in this file: https://github.com/delta-io/delta/blob/5aee6354d167763922276f557ad45152691d1525/spark/src/test/scala/org/apache/spark/sql/delta/DeltaSuite.scala#L1446